.circleci/config.yml

```diff
@@ -64,22 +67,8 @@ setup_venv: &setup_venv
         which pip
         pip install --upgrade pip
 
-install_dep_171: &install_dep_171
```

Comment:
nit: can we add that we are updating what we support in the PR description?

Reply:
done.